api/src/main/java/uk/gov/hmcts/payment/api/controllers/CardPaymentController.java

```diff
@@ -232,4 +237,24 @@ public String return403(AccessDeniedException ex) {
         return ex.getMessage();
     }
 
+    private boolean validateReturnUrl(String returnUrl) throws URISyntaxException {
```

Comment:
Hi @AyanDuttaHMCTS - Can we move this out of controller if possible please?

Reply:
Good Catch, Let me check if we can move this to some util or validator class.

Reply:
Changes Done